airflow/www_rbac/views.py

```diff
@@ -2567,6 +2600,32 @@ def action_set_success(self, drs, session=None):
             flash('Failed to set state', 'error')
         return redirect(self.get_default_url())
 
+    @action('set_canceled', "Set state to 'canceled'",
```

Comment:
i don't see how we hit this endpoint?

Reply:
This is used automagically by the view. I think the decorator adds it into the view. Check it out on stage!